rsrcdump/jsonio.py

```diff
@@ -57,6 +57,7 @@ def resource_fork_to_json(
         res_dirpath = os.path.join(outpath + "_resources", res_dirname)
 
         for res_id, res in res_dir.items():
+            assert isinstance(res_id, (int, bytes))
```

Comment:
Per the inner dict in ResourceFork.tree, `res_dir` is a `dict[int, Resource]`
So, `res_id` is always an int here; bytes should not be possible

Reply:
Fixed in 0e6322